fabfile/adage_server.py

```diff
@@ -210,6 +210,7 @@ def init_instance():
 def build_interface():
     """ have grunt perform a deployment build for us """
     with cd(CONFIG['interface_dir']), \
+            shell_env(PHANTOMJS_BIN='/usr/local/bin/phantomjs'), \
```

Comment:
I was getting problems with this step during AWS deployment, so in an earlier step (under the `adage-deploy` repo actually... will be a separate PR) I am explicitly installing `phantomjs` using `apt-get` and this is where it lands.

Thanks for pointing out the local installation under `node_modules`, @dongbohu. If that one works okay I'll switch to that path.